src/common/Header/Navigation/index.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { toPeople, toMovies } from "../../../routes";
+import { List, StyledNavLink } from "./styled";
+
+export default () => {
+    return (
+        <nav>
+            <List>
+                <StyledNavLink activeClassName="link-active" to={toMovies()}>Movies</StyledNavLink>
+                <StyledNavLink activeClassName="link-active" to={toPeople()}>People</StyledNavLink>
```

Comment:
Tutaj nie musimy przekazywać za każdym razem atrybutu activeClassName, to mamy załatwione w styled-components, dzięki wykorzystaniu metody attr i tam przekazaniu atrybutu ;)

Reply:
racja